pkg/kubelet/nodelease/controller_test.go

```diff
@@ -282,3 +282,111 @@ func TestRetryUpdateLease(t *testing.T) {
 		})
 	}
 }
+
+func TestUpdateUsingLatestLease(t *testing.T) {
+	node := &corev1.Node{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "foo",
+			UID:  types.UID("foo-uid"),
+		},
+	}
+
+	gr := schema.GroupResource{Group: "v1", Resource: "lease"}
+	notFoundErr := apierrors.NewNotFound(gr, "foo")
+	cases := []struct {
+		desc                       string
+		latestLease                *coordinationv1.Lease
+		updateReactor              func(action clienttesting.Action) (bool, runtime.Object, error)
+		getReactor                 func(action clienttesting.Action) (bool, runtime.Object, error)
+		createReactor              func(action clienttesting.Action) (bool, runtime.Object, error)
+		onRepeatedHeartbeatFailure func()
```

Comment:
it seems to be nil in all cases - please remove it

Reply:
done